### ISSUE TITLE
It’s not possible to ser fan speed

### ISSUE THREAD

**Issue by Derbyb** [angoyd/freshintelliventHacs#38]
Thank you for your work with this integration.

It is not possible to change the constant speed.

The speed of the fan doesn’t  change when I change the “constant speed”.

It is still possible to turn the fan on and off but when change the speed (RPM) in constant mode the speed of the fan doesn’t change. And after next update the value in the box goes back to default, 800 RPM.

1) the fan is on at 800 RPM
2) I change speed to 1300 RPM 
3) nothing happens, speed still 800 RPM
4) automatically update 
4) the value goes back to 800 RPM 

If you can look into this it would be fantastic. Thank you.

**Comment by angoyd** [comment]
Just one check, you have set up the integration with authentication key else the fan will not allow changes only reading. I will test i my local test environment too.

**Comment by Derbyb** [comment]
Yes, I used the authentication method. It’s is possible for me to turn off and on the fan from the integration but not to set the speed.

Maybee, just maybee its the IPhone App (from fresh) that interferes with the integration!? As I recall the official app from fresh was running (on my IPhone) when I tried the integration (in background).

**Comment by angoyd** [comment]
Great, I will test to see if I can replicate it.

**Comment by angoyd** [comment]
I have made some inital tests now I can change the RPM value, but the sensor showing the RPM value isnt updated directly, can you try the following. No app on phone open, and verify that the can is operating at constant speed ( in mode) , then change the RPM and wait about 2 minutes ( depending of the update freq you have setup in the config) and see if it works. This works for me.
